Source/Scene/Cesium3DTileFeature.js

```diff
@@ -104,6 +104,20 @@ define([
     });
 
     /**
+     * Returns an array of property names for the feature.
+     * <p>
+     * {@link Cesium3DTileFeature#show} and {@link Cesium3DTileFeature#color} are not equivalent to
+     * <code>'show'</code> and <code>'color'</code> properties; the former are runtime-specific properties
+     * that are not part of the feature's properties in the stored 3D Tileset.
+     * </p>
+     *
+     * @returns {String[]} The names of the feature's properties.
+     */
+    Cesium3DTileFeature.prototype.getPropertyNames = function() {
```

Comment:
The main difference will be that `batch_id` will be sent into `Cesium3DTileBatchTable.getPropertyNames` and that function will need to account for the hierarchy. This is fine as it is now though.